index.html

```diff
@@ -293,4 +350,5 @@ <h3 class="readmore" href="#">Read more</h3>
             </div>
         </footer>
     </body>
+>>>>>>> develop
```

Comment:
Čia konfliktas likęs. Dėl to duplikuojasi sekcija
![image](https://user-images.githubusercontent.com/50375573/103783064-eb418200-5040-11eb-9beb-f44a2baabd7d.png)

Reply:
nesuprantu kaip, nes pas mane tai gera rodo viska.
![image](https://user-images.githubusercontent.com/74533072/103787712-9acd2300-5046-11eb-9498-19848faf4075.png)

Reply:
Bandom dabar